libtransmission/session.cc

```diff
@@ -75,6 +75,9 @@ static auto constexpr DefaultPrefetchEnabled = bool{ true };
 #endif
 static auto constexpr SaveIntervalSecs = int{ 360 };
 
+static void bandwidthGroupRead(tr_session* session, char const* configDir);
+static int bandwidthGroupWrite(tr_session* session, char const* configDir);
```

Comment:
prefer `std::string_view` for configDir in these two unless there's a reason it needs to be `char const*`

Reply:
https://github.com/transmission/transmission/pull/2783